src/components/minting/SelectCoCreators.tsx

```diff
@@ -137,6 +139,30 @@ const FormContent: FunctionComponent<FormContentProps> = ({
         </Button>
       </Stack>
 
+      <Stack mt={ 1.5 } flexDirection="row">
+        <Typography variant="subtitle2">
+          For every additional collaborator who will receive royalties, the fee
+          to complete the minting process will increase by 1.5 ADA.
```

Comment:
Saw a little discussion back and forth on the CU task and Slack but wanted to get clarity on this. This 1.5 ADA amount isn't set right? Do we have a scheme for these values to be updated? Or is that only happening on a task for integrating the Minting backend process?

Reply:
Correct, it's not set. They are still trying to figure it out and how the collaborators affect the pricing from what i recall. Either way this should be small enough to implement at a later stage once we know the math part